api/v1alpha1/openapi.yaml

```diff
@@ -4192,15 +4198,24 @@ components:
         approved:
           type: boolean
           description: Indicates whether the request has been approved.
-        approvedBy:
-          type: string
-          description: The name of the approver.
-        approvedAt:
-          type: string
-          format: date-time
-          description: The time at which the request was approved.
       required:
         - approved
+    EnrollmentRequestApprovalStatus:
+      allOf:
+      - $ref: "#/components/schemas/EnrollmentRequestApproval"
+      - type: object
+        description: EnrollmentRequestApprovalStatus contains information about the status of approval of a device enrollment request.
+        properties:
+          approvedBy:
+            type: string
+            description: The name of the approver.
+          approvedAt:
+            type: string
+            format: date-time
+            description: The time at which the request was approved.
+        required:
+        - approvedBy
+        - approvedAt
```

Comment:
What's the motivation of splitting the `approvedBy` and `approvedAt` out of the `EnrollmentRequestApproval`? And not the approved labels, for example?

Reply:
The motivation is to have a cleaner API.
Before the change we used the same object (`EnrollmentRequestApproval`) for incoming approval requests and also for ER status. However the original `EnrollmentRequestApproval` contained `approvedBy` and `approvedAt` fields which do not make sense for the approval request, only for status.

So I split them to a separate objects - one for approval `EnrollmentRequestApproval` and one for ER status `EnrollmentRequestApprovalStatus`.